scss/upload/_layout.scss

```diff
@@ -139,60 +147,58 @@
             white-space: nowrap;
             position: relative;
         }
+
         .k-upload-status {
-            display: flex;
-            align-items: center;
-            position: relative;
-            margin-left: auto;
+            position: absolute;
+            line-height: 1em;
+            right: $padding-x;
+            top: $padding-x + $padding-y;
```

Comment:
Why does horizontal padding influence the `top` position? Shouldn't that be `(multiplier) * $padding-y`, by any chance?